contents/docs/session-replay/ios.mdx

```diff
@@ -58,6 +58,7 @@ imvProfilePhoto.accessibilityIdentifier = "ph-no-capture"
 
 - When using the [SwiftUI TextField](https://developer.apple.com/documentation/swiftui/textfield) with [UITextInputTraits](https://developer.apple.com/documentation/uikit/uitextinputtraits), setting traits such as `TextField("Email", text: ...).keyboardType(.emailAddress)` will always be automatically masked since it's for private text.
 - The [SwiftUI SecureField](https://developer.apple.com/documentation/swiftui/securefield) view is always automatically masked.
+- You can manually mark a SwiftUI View for masking using the `postHogMask(_:)` view modifier
```

Comment:
Can we add a code snippet similar to how we do with the `accessibilityIdentifier` for UIKit? that's easier for novice developers